src/main/java/com/sprint/mission/discodeit/entity/Message.java

```diff
@@ -1,64 +1,68 @@
 package com.sprint.mission.discodeit.entity;
 
-import lombok.Getter;
+import jakarta.persistence.*;
+import lombok.*;
+
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
 
-import java.io.Serializable;
-import java.time.Instant;
 import java.util.ArrayList;
 import java.util.List;
-import java.util.UUID;
 
+
+@Entity
 @Getter
-public class Message implements Serializable {
-    private static final long serialVersionUID = 1L;
-    private final UUID id;
-    private final Instant createdAt;
-    private Instant updatedAt;
+@Setter(AccessLevel.PACKAGE)
+@Table(name = "messages")
+@NoArgsConstructor
+public class Message extends BaseUpdatableEntity {
+
+    @Column
     private String content;
-    private final UUID channelId;
-    private final UUID authorId;
-    private final List<UUID> attachmentIds;
 
+    @ManyToOne(cascade = CascadeType.ALL, fetch = FetchType.LAZY)
```

Comment:
ManyToOne 관계에서 `CascadeType.ALL ` 설정 적용하는건 매우 위험하기에 제거하는게 좋을 것 같아요.
현재 상태에서 Channel-Message은 부모-자식  관계인데요,  

이때 ManyToOne에 `CascadeType.ALL ` 설정 적용하면 Message가 삭제될 때 Channel도 같이 삭제되어요. (의도치 않은 동작)

```suggestion
    @ManyToOne(fetch = FetchType.LAZY)
```

- 참고 문서: [영속성 전이(Cascade) ManyToOne 시 주의할점 - 상위 엔티티 삭제 문제](https://0soo.tistory.com/133)